working/manual_changes/2022/2022-02-28.sql

```diff
@@ -0,0 +1,441 @@
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'{Ct_value}'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'milliliter per pound (US)',
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'mL/[lb_us]'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'millibar',
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'mbar'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'nanogram per nanogram',
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'ng/ng'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'free thyroxine index',
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'{FTI%}'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'milligram per deciliter per 24 hours',
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'mg/dL/(24.h)'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'picogram per milligram of creatinine',
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'pg/mg{creat}'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'picomole per hour per microliter',
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'pmol/hr/uL'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'nanomole per 24 hours per milligram',
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'nmol/(24.h)/mg'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'milliliter per millibar',
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'ml/mbar'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'millibar per centimeter of water',
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'ml/cm [H2O]'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'millibar per liter',
+pDomain_id =>'Unit',
+pVocabulary_id =>'UCUM',
+pConcept_class_id =>'Unit',
+pStandard_concept =>'S',
+pConcept_code =>'mbar/L'
+);
+END $_$;
+
+DO $_$
+BEGIN
+PERFORM vocabulary_pack.AddNewConcept(
+pConcept_name =>'fraction',
```

Comment:
Unclear, can we use ratio here?

Reply:
Discussed with Irina - agreement on 'fraction' achieved inside the team before adding this unit